test/utils/execute_helpers.hpp

```diff
@@ -5,12 +5,20 @@
 #pragma once
 
 #include "execute.hpp"
+#include <initializer_list>
 
 namespace fizzy::test
 {
-inline execution_result execute(const Module& module, FuncIdx func_idx, std::vector<uint64_t> args)
+inline execution_result execute(
+    Instance& instance, FuncIdx func_idx, std::initializer_list<uint64_t> args)
```

Comment:
Does it not help in tests where you declare args as arrays?

Reply:
I did not included this overload in end_to_end tests where it has small number of case that could be converted to `std::array`.

Reply:
It's 4 cases, I'd still use it.